rocketmq-common/src/common/namesrv/namesrv_config.rs

```diff
@@ -147,6 +149,104 @@ impl NamesrvConfig {
         Self::default()
     }
 
+    pub fn get_all_configs_format_string(&self) -> Result<String, String> {
+        let mut json_map = HashMap::new();
+
+        json_map.insert(
```

Comment:
This can be improved if we replace `#[serde(alias = "property")]` with `#[serde(rename = "property")]`.
Then we can use serde_json here, but I am not sure how many things this change would break.